containers/Containerfile.ci

```diff
@@ -0,0 +1,91 @@
+FROM registry.fedoraproject.org/fedora:34
+
+MAINTAINER "Candlepin Team" <candlepin@redhat.com>
+# should be the UID of whatever user is running podman, change this with: podman build --build-arg UID="$(id -u)"
+ARG UID=1020
+ENV SMDEV_CONTAINER_OFF='True'
+
+RUN dnf install --disablerepo=updates -y \
+    cmake \
+    dbus-daemon \
+    dnf-utils \
+    gcc \
+    gettext \
+    git \
+    intltool \
+    json-c-devel \
+    langpacks-as \
+    langpacks-bn \
+    langpacks-de \
+    langpacks-es \
+    langpacks-fr \
+    langpacks-gu \
+    langpacks-hi \
+    langpacks-it \
+    langpacks-ja \
+    langpacks-kn \
+    langpacks-ko \
+    langpacks-ml \
+    langpacks-mr \
+    langpacks-or \
+    langpacks-pa \
+    langpacks-pt_BR \
+    langpacks-ru \
+    langpacks-ta \
+    langpacks-te \
+    langpacks-zh_CN \
+    langpacks-zh_TW \
```

Comment:
these langpacks should not be needed anymore now, after #2885

Reply:
thanks! done.